mcode/request-generator
# 672 request generator sends auth number in new rx

## Describe your changes

When send new rx button is clicked, the auth number gets added in to the newRx message. 

To test:
- Select a medication (turalio - john snow). Hit sign order and fill out all of the forms. 
- When you hit rems-etasu you'll see the auth number is returned in its parameters. 
- Go back and select 'send rx to pharmacy'. Check the newRx message from the networks tab or the console and see in the header next to case number, an AuthorizationNumber gets sent. A future ticket (673) is for the PIMS side to handle the auth number


## Issue ticket number and Jira link

 https://jira.mitre.org/browse/REMS-672 

## Checklist before requesting a review

## Checklist for conducting a review

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.
